pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -174,6 +177,32 @@ func TestGetRemoteEntrypoint(t *testing.T) {
 	}
 }
 
+func TestEntrypointCacheLRU(t *testing.T) {
+	entrypoint := []string{"/bin/expected", "entrypoint"}
+	entrypointCacheSize = 32
+	entrypointCache, err := NewCache()
+	if err != nil {
+		t.Errorf("couldn't new entrypoint cache: %v", err)
+	}
+
+	for i := 0; i < 64; i++ {
```

Comment:
how about just one past the size of the cache, instead of 64? e.g. `for i := 0; i < entrypointCacheSize + 1; i++ `

or if 64 is a better choice, we can use a `const` or variable for the value, e.g.

```
const (
    cacheSize = 32
    exceedCacheSize = cacheSize * 2
)
```